hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcServer.java

```diff
@@ -68,7 +68,10 @@
    */
   public static final String HBASE_NETTY_EVENTLOOP_RPCSERVER_THREADCOUNT_KEY =
     "hbase.netty.eventloop.rpcserver.thread.count";
+  public static final String HBASE_NETTY_EVENTLOOP_RPCSERVER_BOSSTHREADCOUNT_KEY =
```

Comment:
Remember that I was there when I went through the hbase netty code, let me bring up my memory and come back.